docs/tutorial/persistence.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Persistence
+id: persistence
+---
+# Persistence
+
+## Why content slowly disappears from Swarm
+The storage capacity of the whole Swarm network equals the the sum of the capacity of all connected nodes. As nobody is restricted to upload content to Swarm, eventually the storage capacity of the Swarm network reaches its limits and nodes face the choice to either stop accepting new chunks or delete content which is there already for a while. 
+
+As Swarm nodes are programmed to behave economically, they won't stop accepting new chunks. The reasoning for this is that newly uploaded chunks are most likely to be accessed, offering the node an opportunity for profit. Instead, they will delete some chunks to make space for the new ones. The chunks that are deleted are the ones which were uploaded or accessed furthest away in the past.
+
+:::info
+The garbage collection queue deletes chunks which were uploaded or **accessed** furthest away in the past. This means that you can persist your chunks by regularly accessing it (or making it super duper awesome, so other people access it)
+:::
+
+## Persistence on your own node by local pinning
+It is possible to instruct your node never to delete certain chunks, by locally pinning the chunks. Locally pinning chunks means that you will always be able to access these chunks via your node. It does mean that those chunks will be available in the network.
```

Comment:
**However** It **doesn't** mean that those chunks will **always** be available in the network.

Reply:
Thx